classes/decorator.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './nameable'
+class Decorator
```

Comment:
- [ ] It will be great to ensure that your decorator class inherits from the Namable class. You forgot to add this. Example ⬇️ 
`class Decorator < Nameable`